mwclient/client.py

```diff
@@ -262,9 +288,9 @@ def split_num(s):
                     break
                 i += 1
             if s[i:]:
-                return (int(s[:i]), s[i:], )
+                return (int(s[:i]), s[i:],)
             else:
-                return (int(s[:i]), )
+                return (int(s[:i]),)
 
         version_tuple = sum((split_num(s) for s in version), ())
```

Comment:
I just wasted fifteen minutes figuring out what the hell this does...it flattens the sequence of tuples into one tuple...probably would be more idiomatic to do `tuple(itertools.chain(split_num(s) for s in version)))`...

Reply:
The code has been removed in #359 🚀.